lib/model/occupation/Task.dart

```diff
@@ -0,0 +1,64 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+import 'dart:convert';
+
+enum JobCategory {
+  core,
+  supplemental,
+  notAvailable,
+}
+
+class Task {
```

Comment:
I'm not seeing this class being used anywhere yet, but it looks like `Occupation` could include a list of tasks.

Reply:
Yes we added this now since we will probably need this for future data classes.